docs/example_workflow_explained.rst

```diff
@@ -0,0 +1,316 @@
+``bmorph`` Example Workflow
```

Comment:
We should maybe consider renaming this to something descriptive like `10 minute tutorial: Getting your first bias corrections with bmorph`

Reply:
Renamed the header, but w/o the '10 minute' part. I'll also rename the file as `bmorph_tutorial`